conf/messages.en

```diff
@@ -243,7 +243,7 @@ start.addNow = Add now
 taxResidencyCountry.title = What country is the platform operator tax resident in?
 taxResidencyCountry.heading = What country is {0} tax resident in?
 taxResidencyCountry.checkYourAnswersLabel = What country is {0} tax resident in?
-taxResidencyCountry.error.required = Enter the country {0} is tax resident in
+taxResidencyCountry.error.required = Start typing in country and choose from the list
```

Comment:
since you have removed the param passed into this messge key pair, then will also need to update:
https://github.com/hmrc/digital-platform-reporting-operator-frontend/blob/main/app/forms/TaxResidencyCountryFormProvider.scala